bot/seasons/evergreen/issues.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Issues(commands.Cog):
+    """Cog that allows users to retrieve issues from GitHub."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=("issues",))
+    async def issue(self, ctx, number: int, repository: str = "seasonalbot", user: str = "python-discord"):
+        """Command to retrieve issues from a GitHub repository."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
+        failed_status = {
+            404: f"Issue #{number} doesn't exist in the repository {user}/{repository}.",
+            403: f"Rate limit exceeded. Please wait a while before trying again!"
+        }
+
+        async with self.bot.http_session.get(url) as r:
+            json_data = await r.json()
+            response_code = r.status
+
+        if response_code in failed_status:
+            return await ctx.send(failed_status[response_code])
+
+        issue_embed = discord.Embed(colour=0x00ff37)
+        issue_embed.add_field(name="Repository", value=f"{user}/{repository}", inline=False)
+        issue_embed.add_field(name="Issue Number", value=f"#{number}", inline=False)
+        issue_embed.add_field(name="Status", value=json_data["state"].title())
+        issue_embed.add_field(name="Link", value=url, inline=False)
+
+        description = json_data["body"]
+        if len(description) > 1024:
+            truncation_message = "** ...\n\nContent truncated, please follow the link for more!**"
```

Comment:
I am imagining it would look strange to have the ellipses be bold. I also don't think there should be a space before them.

```suggestion
            truncation_message = "...\n\n**Content truncated, please follow the link for more!**"
```

Reply:
I think it looks worse without the space

![image](https://user-images.githubusercontent.com/5323929/61676821-55d6f480-accb-11e9-9291-bc98a670f5df.png)
![image](https://user-images.githubusercontent.com/5323929/61676829-5c656c00-accb-11e9-9221-9fad816affec.png)
![image](https://user-images.githubusercontent.com/5323929/61676844-625b4d00-accb-11e9-8ed8-0303485fa18c.png)

Reply:
Thanks for the images. I still like it without a space, but I'm not gonna block the PR with this. I've already approved so do as you wish.